src/styles/index.scss

```
@@ -1,137 +1,33 @@
-$spotify-green: rgb(30, 215, 96);
-
-/* Global styles*/
-
 body {
-  font-family: 'montserrat', sans-serif;
+  font-family: "montserrat", sans-serif;
   margin: 0;
   font-size: 16px;
   line-height: 1.5;
 
   background: #000;
```

Stylelint (17.11.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)